src/Collections/Sequences/LittleEndianNat.dfy

```diff
@@ -56,7 +56,7 @@ abstract module LittleEndianNat {
   }
 
   /* Given the same sequence, ToNatRight and ToNatLeft return the same nat. */
-  lemma LemmaToNatLeftEqToNatRight(xs: seq<uint>)
+  lemma {:vcs_split_on_every_assert} LemmaToNatLeftEqToNatRight(xs: seq<uint>)
```

Comment:
Since this has no effect on 3.3, we should make this PR bump Dafny up to 3.4 at the same time to be a good atomic change.

Reply:
Will do. And I'll also update one other test that's failing with 3.4 since the error message now has slightly more information in it.